commit 3e5a5b0d31a08b97e7dcd9c03e289d7a75309fad (refs/changes/42/742/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-19T15:09:16-07:00 (3 years ago)
    
    PUBAPI-1337 Update node version for CVE-2016-5180
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/Makefile b/Makefile
index 753d0fa..b26eded 100644
--- a/Makefile
+++ b/Makefile
@@ -47,7 +47,7 @@ CLEAN_FILES	+= node_modules cscope.files
 
 # The prebuilt sdcnode version we want. See
 # "tools/mk/Makefile.node_prebuilt.targ" for details.
-NODE_PREBUILT_VERSION=v0.10.47
+NODE_PREBUILT_VERSION=v0.10.48
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_IMAGE=de411e86-548d-11e4-a4b7-3bb60478632a
 	NODE_PREBUILT_TAG=zone
